lib/teiserver/autohost/tachyon_handler.ex

```diff
@@ -20,7 +20,10 @@ defmodule Teiserver.Autohost.TachyonHandler do
 
   @type start_response :: %{
           ips: [String.t()],
-          port: integer()
+          port: integer(),
+          engine: %{String.t() => String.t()},
+          game: %{String.t() => String.t()},
+          map: %{String.t() => String.t()}
```

Comment:
Although these types are valid they are not great. You should prefer using known keys with atoms like so: `%{version: String.t()}`

Reply:
Done